### PR TITLE
Changed String to AbstractString at FreeTypeAbstraction:20

### PR DESCRIPTION
**Pull request by hustf**
Fix deprecation warning.

### CODE REVIEW

File: src/FreeTypeAbstraction.jl [hunk 1]
```diff
@@ -17,7 +17,7 @@ immutable FontExtent{T}
     scale               ::Vec{2, T}
 end
 immutable FontFace
-    name::String
+    name::AbstractString
```
Comment (SimonDanisch) [inline src/FreeTypeAbstraction.jl]: Hm... I think this should either be:
```Julia
 immutable FontFace
     name::Compat.String
end
```
or
```Julia
 immutable FontFace{T<:AbstractString}
     name::T
end
```

**Comment by hustf** [comment]
Now, I realize such small commits may be more of a hazzle, but there's a barrier to break. Last time I checked this package I saw a number of small potential improvements, and didn't do anything about it.

**Comment by SimonDanisch** [comment]
No, this is great :) That was definitely an oversight by me!

**Comment by SimonDanisch** [comment]
thanks!

**Pull request merged**